charts/celeborn/values.yaml

```diff
@@ -34,6 +34,8 @@ masterReplicas: 3
 workerReplicas: 5
 
 securityContext:
+  runAsUser: 10006
+  runAsGroup: 10006
```

Comment:
why them are required?

Reply:
We define Celeborn user in Dockerfile (uid: 10006 & gid: 10006), the call is shown here and can be configured as a variable by the user